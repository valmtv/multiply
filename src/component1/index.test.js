import React from 'react';
import { shallow } from 'enzyme';

import Component1 from './';

describe('snapshots', () => {
  it('renders correclty', () => {
    const wrapper = shallow(
      <Component1 />,
    );

    expect(wrapper).toMatchSnapshot();
  });
});
