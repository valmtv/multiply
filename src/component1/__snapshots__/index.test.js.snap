// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshots renders correclty 1`] = `
<div>
  Component1
</div>
`;
